pkg/apiserver/authentication/internaloauth/tokenauthenticator_test.go

```diff
@@ -0,0 +1,383 @@
+package internaloauth
```

Comment:
Same.

Reply:
> Same.

No.  The previous one part of a giant test file.  There is no previous file to move.